src/components/form/examples/form-with-help.tsx

```diff
@@ -0,0 +1,47 @@
+import { Component, h, State } from '@stencil/core';
+import { schema } from './help-form-schema';
+
+/**
+ * Form elements with contextual help
```

Comment:
What's contextual about it?

To me contextual suggests that the help would somehow (dynamically) depend on the context, whatever context means here. So a helper text that depends on validation errors is _more_ contextual than this piece of help. This is just static info providing rich text "help".

Or did you mean to say that the help provides context?

Reply:
I didn't have any specific intentions. Maybe was just thinking that a form field in the context of that form, needs some further explanation / help :) We can change this heading.

Reply:
> a form field in the context of that form, needs some further explanation

So it's providing help per form field then. When you say "the context of that form", I'm just getting more confused (I don't even know what form means here. The form design described by the schema?) 😄